subprojects/docs/src/snippets/java/toolchain-management/groovy/settings.gradle

```diff
@@ -1,15 +1,18 @@
+plugins {
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.2'
+}
+
 import java.util.Optional
 import javax.inject.Inject
 
-apply plugin: AzulPlugin
 apply plugin: AdoptiumPlugin
 
 // tag::toolchain-management[]
 toolchainManagement {
     jvm { // <1>
         javaRepositories {
-            repository('azul') { // <2>
-                resolverClass = AzulResolver
+            repository('disco') { // <2>
+                resolverClass = org.gradle.toolchains.foojay.FoojayToolchainResolver
                 credentials {
```

Comment:
We should move that to the `adoptium` element, as it makes no sense for our foojay plugin

Reply:
You mean the authentication bits, right? Moving it.